spec/services/data_requests/write_user_info_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+describe DataRequests::WriteUserInfo do
+  describe '#call' do
+    it 'writes a file with user information' do
+      user_report = JSON.parse(
+        File.read('spec/fixtures/data_request.json'), symbolize_names: true
+      ).first
+
+      Dir.mktmpdir do |dir|
+        described_class.new(user_report, dir).call
+        user = File.read(File.join(dir, 'user.csv'))
+        headings = user.split("\n\n").map do |info_group|
+          info_group.split("\n").first
+        end
```

Comment:
I think we can clean this up a tad too


```suggestion
        headings = CSV.read(File.join(dir, 'user.csv'), headers: true).headers
```

Reply:
oh wait I just realized the current code is grabbing the first column not the first row 🤦

Reply:
it's unfortunate because the file says "csv", but it is not in fact a CSV.

Reply:
can we/should we change the file extension to `.txt`?

Reply:
Yep, we probably should. I'd defer to @jmhooper on it though as there may be a requirement for this format? It may also require updating documentation elsewhere (maybe handbook?) on the usage of `data_request` tasks.

Reply:
Yeah, the agents request CSVs for the artifacts we send them because they open everything in MS Excel.